docs: add from_cfn section in blogpost v1.25 (#4356)
aws/copilot-cli: site/content/blogs/release-v125.en.md

## Environment Addons
### Import Values From CloudFormation Stacks In Workload Manifests

You can now import values from environment addons CloudFormation stack or any other arbitary stack in your workload manifest using `from_cfn`.

> To reference a value from another CloudFormation stack, users should first export the output value from the source stack.

Reviewer: Can we have an example of what the `Export` should look like, and a link to CFN doc that talks about `Export`?

Author: added the example and link to doc

## Environment Addons
### Import Values From CloudFormation Stacks In Workload Manifests

> You can now import values from environment addons CloudFormation stack or any other arbitary stack in your workload manifest using `from_cfn`.

Reviewer 2: ```suggestion
You can now import values from environment addons' CloudFormation stacks or any other stack in your workload manifest using `from_cfn`.
```

To reference a value from another CloudFormation stack, users should first export the output value from the source stack.

> Here is an example on how the `Outputs` section of a CloudFormation template looks when exporting values from other stacks or creating cross-stack references.

Reviewer 2: ```suggestion
Here is an example of how the `Outputs` section of a CloudFormation template looks when exporting values from other stacks or creating cross-stack references.
```

> Here is an example on how the `Outputs` section of ClouFormation template looks like to export values from other stacks or to create a cross-stack references.

Reviewer 2: ```suggestion
Here is an example on how the `Outputs` section of a CloudFormation template looks when exporting values from other stacks or creating cross-stack references.
```

```yaml
Outputs:
  WebBucketURL:
    Description: URL for the website bucket
    Value: !GetAtt WebBucket.WebsiteURL
    Export:
      Name: stack-WebsiteUrl # <- Unique export name within the region.
```

> To find our more, you can see [this page](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/outputs-section-structure.html) 

Reviewer 2: ```suggestion
To find our more, see [this page].(https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/outputs-section-structure.html) 
```
